MultisigIntegrationTests/CoreData/NetworkTests.swift

```diff
@@ -197,6 +197,40 @@ class NetworkTests: CoreDataTestCase {
             themeBackgroundColor: "#000000")
     }
 
+    func test_networkSafes() throws {
```

Comment:
Missing test cases:
- when no safes exist, then should return empty
- when all safes in the same network, should do the correct ordering

Maybe unrelated, but it's good if we fix it here:
- when multiple safes with different network and same address exist, then works fine
- when last safe removed from the network, then selects next safe in order

Reply:
> when no safes exist, then should return empty
added 

> when all safes in the same network, should do the correct ordering
this is already covered

> when multiple safes with different network and same address exist, then works fine
added

> when last safe removed from the network, then selects next safe in order
I will add it in other already open PR